src/ralph/admin/static/auto-complete-helpers.js

```diff
@@ -45,6 +45,15 @@ function dismissAddRelatedObjectPopup(win, newId, newRepr) {
             } else {
                 elem.value = newId;
             }
+        } else if (elemName == 'UL') {
+            // TODO: jezeli brak elementow to nic nie rob
```

Comment:
What? :)

Reply:
I closed this commit and push another. I'll fix it in a minute ;)